### ISSUE TITLE
Bluetooth printing issue

### ISSUE THREAD

**Issue by Reshmank** [srehanuddin/Cordova-Plugin-Bluetooth-Printer#2]
Bluetooth print plugin working correctly in my phonegap app . but bluetooth printer not working . just connecting the device ,but nothg is printed . what will i do , need extra jar file to attach my app , please respond me

**Comment by srehanuddin** [comment]
For using bluetooth printer we need to pair printer manually from phone, then we can print via this plugin.

**Comment by Reshmank** [comment]
yes ,am already paired blutooth printer and mobile ,connected and data sent alert was displayed in our app. but not print any data into bluetooth printer . i think need to add any extra jar file for our plugin

**Comment by srehanuddin** [comment]
No, there is no need of any jar file, only requirement is to install plugin, select printer and then send print, is your printer support POS? 

and please also try to print from printer vender application

**Comment by Reshmank** [comment]
yes all function are working . printer vender applications means ,i did't catch you . can i create a custom function for this plugin ?? , what purpose we use pos ?

**Comment by srehanuddin** [comment]
Yes, 

"my bluetooth printer company sent me the docuemntaion and sample app for printing"

did you tried from this sample app ?

**Comment by Reshmank** [comment]
yes ,its working , they only send app and one jar file only .that is why am asking you . then i add that jar file to my app with bluetooth printer plugin .but not working

**Comment by wonk4rol** [comment]
@Reshmank i have done with this great plugin, you need not use the vendor jar file. Just use this plugin, and it works. :-D

**Comment by pufanalexandru** [comment]
I have the same problem. Data is sent to printer, but it does not print

**Comment by wonk4rol** [comment]
@pufanalexandru you must disconnect printer to print.

**Comment by pufanalexandru** [comment]
Thanks, I solved the issue.

**Comment by wonk4rol** [comment]
@pufanalexandru you are welcome.

**Comment by asiddeen** [comment]
@pufanalexandru how did u solve the issue please?

**Comment by ssarnot** [comment]
@asiddeen and @Reshmank look here: https://github.com/srehanuddin/Cordova-Plugin-Bluetooth-Printer/issues/23

**Comment by asiddeen** [comment]
Thanks @reshmank I manage to make it work long time ago. 👍🏽

**Comment by fmyang123** [comment]
i’m not able to install it.

Error: Failed to fetch plugin https://github.com/srehanuddin/Cordova-Plugin-Bluetooth-Printer.git via registry.
 Probably this is either a connection problem, or plugin spec is incorrect.
 Check your connection and plugin name/version/URL.
 Error: cmd: Command failed with exit code 4294963228 Error output:
 npm WARN package.json testing@1.1.1 No repository field.
 npm WARN package.json testing@1.1.1 No README data
 npm WARN package.json testing@1.1.1 No license field.
 npm ERR! addLocal Could not install C:\Users\user\AppData\Local\Temp\npm-8924-c0b696a6\git-cache-a57a5edad3ef\147da4a81ed239f4ead31268f4971afcef4f0dba
 npm ERR! Windows_NT 6.1.7601
 npm ERR! argv "C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js" "install" "https://github.com/srehanuddin/Cordova-Plugin-Bluetooth-Printer.git" "--save"
 npm ERR! node v4.4.0
 npm ERR! npm v2.14.20
 npm ERR! code EISDIR
 npm ERR! errno -4068
 npm ERR! syscall read

npm ERR! eisdir EISDIR: illegal operation on a directory, read
 npm ERR! eisdir This is most likely not a problem with npm itself
 npm ERR! eisdir and is related to npm not being able to find a package.json in
 npm ERR! eisdir a package you are trying to install.

npm ERR! Please include the following file with any support request:
 npm ERR! C:\Users\user\workspace\testing\node_modules\npm-debug.log
